### ISSUE TITLE
Indicate existence of games with pending turns in the header / go to next game button

### ISSUE THREAD

**Issue by dlpierce** [tobymao/18xx#2532]
When viewing a game, it would be nice to have an indicator in the header div when there are other games waiting for the user to take their turn. This could be expanded to display a menu in the header with direct links to all of the user's active games, with games waiting on the user highlighted.
